core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -739,17 +739,23 @@ public void testLoadMetadataTest() throws Exception {
         LoadMetadataOptions.defaults().setCreateAncestors(true));
 
     // TODO(peis): Avoid this hack by adding an option in getFileInfo to skip loading metadata.
-    mThrown.expect(FileAlreadyExistsException.class);
-    mFileSystemMaster
-        .createDirectory(new AlluxioURI("alluxio:/a"), CreateDirectoryOptions.defaults());
+    try {
+      mFileSystemMaster
+          .createDirectory(new AlluxioURI("alluxio:/a"), CreateDirectoryOptions.defaults());
+      Assert.fail();
```

Comment:
nit: add message to fail: `Assert.fail("createDirectory was expected to fail with FileAlreadyExistsException")`